### PR TITLE
feature(ui-ux): vaults card 2.0

### PR DESCRIPTION
**Pull request by chloezxyy**
#### Which issue(s) does this PR fixes?:

Fixes #

#### Additional comments?:
- [x] Some missing translations on Vault Card 2.0
- [x] Hid the circular progress bar on web because it's throwing error

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [ ] Added e2e tests*
- [x] Added translations*

### CODE REVIEW

File: mobile-app/app/screens/AppNavigator/screens/Loans/components/PriceOracleInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      <TouchableOpacity
+        onPress={props.onPress}
+        style={tailwind("text-center flex-row items-center")}
+      >
+        <ThemedTextV2
+          dark={tailwind("text-mono-dark-v2-500")}
+          light={tailwind("text-mono-light-v2-500")}
+          style={tailwind("font-normal-v2 text-sm mr-1")}
+        >
+          {translate("", "Loan tokens get their prices from oracles.")}
```
Comment (pierregee) [inline mobile-app/app/screens/AppNavigator/screens/Loans/components/PriceOracleInfo.tsx:22]: no translations yet?

**Review by pierregee — CHANGES_REQUESTED** [review]

File: package-lock.json [hunk 2]
```diff
@@ -8844,6 +8844,22 @@
         "react-native-safe-area-context": ">= 3.0.0"
       }
     },
+    "node_modules/@react-navigation/material-top-tabs": {
```
[the remark this answers is not in the document]
Comment (chloezxyy) [reply package-lock.json:8847]: reverted

File: mobile-app/app/components/SymbolIcon.tsx [hunk 3]
```diff
@@ -8,3 +8,11 @@ export function SymbolIcon(props: {
   const Icon = getNativeIcon(props.symbol);
   return <Icon width={16} height={16} {...props.styleProps} />;
 }
+
+export function SymbolIconV2(props: {
```
[the remark this answers is not in the document]
Comment (chloezxyy) [reply mobile-app/app/components/SymbolIcon.tsx:12]: Removed SymbolIconV2 and added props for icon sizing in SymbolIcon

File: mobile-app/app/components/TokenIconGroup.tsx [hunk 4]
```diff
@@ -66,3 +66,52 @@ export function TokenIconGroup(props: TokenIconGroupProps): JSX.Element {
     </View>
   );
 }
+
+export function TokenIconGroupV2(props: TokenIconGroupProps): JSX.Element {
```
[the remark this answers is not in the document]
Comment (chloezxyy) [reply mobile-app/app/components/TokenIconGroup.tsx:70]: We will standardize the overlap the way we did for loans

File: mobile-app/app/screens/AppNavigator/screens/Loans/components/VaultCardV2.tsx [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+  onBottomSheetLoansTokensListSelect: ({
+    onPress,
+    loanTokens,
+  }: {
+    onPress: (item: LoanToken) => void;
+    loanTokens: LoanToken[];
+  }) => void;
+}
+
+export function VaultCardV2(props: VaultCardProps): JSX.Element {
```
Comment (pierregee) [inline mobile-app/app/screens/AppNavigator/screens/Loans/components/VaultCardV2.tsx:50]: image
1. Borrow Button is wrong color, will it be part of the circular lib PR?
2. Spacing is also off

Comment (pierregee) [reply mobile-app/app/screens/AppNavigator/screens/Loans/components/VaultCardV2.tsx:50]: 3. When using $, it should be 2 decimal places

Comment (pierregee) [reply mobile-app/app/screens/AppNavigator/screens/Loans/components/VaultCardV2.tsx:50]: 4. Liquidated Vault collateral is not displaying
5. Liquidated Vault color is always black
image

File: mobile-app/app/screens/AppNavigator/screens/Loans/components/BottomSheetLoanTokensList.tsx [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+import { translate } from "@translations";
+import { memo } from "react";
+import * as React from "react";
+import { NumericFormat as NumberFormat } from "react-number-format";
+import { getPrecisedTokenValue } from "../../Auctions/helpers/precision-token-value";
+import { getActivePrice } from "../../Auctions/helpers/ActivePrice";
+import { TokenIcon } from "../../Portfolio/components/TokenIcon";
+import { TokenNameText } from "../../Portfolio/components/TokenNameText";
+
+export const BottomSheetLoanTokensList = ({
```
Comment (pierregee) [inline mobile-app/app/screens/AppNavigator/screens/Loans/components/BottomSheetLoanTokensList.tsx:18]: image
Is there a figma for this that you followed? e.g other bottomsheets I see is that the header is aligned with the close (x) button

**Review by pierregee — CHANGES_REQUESTED** [review]

File: mobile-app/app/screens/AppNavigator/screens/Loans/components/VaultCardV2.tsx (hunk 5 above)
Comment (chloezxyy) [reply mobile-app/app/screens/AppNavigator/screens/Loans/components/VaultCardV2.tsx:50]: Point 1,2,3,4 and 5 are done on Joshua's PR :D

**Pull request merged**